binaries/build.py

```diff
@@ -29,7 +28,7 @@ def build_dist_whl(args):
         create_wheel_cmd = "python setup.py "
     else:
         print("## Started torchserve, model-archiver and workflow-archiver build")
-        create_wheel_cmd = "python setup.py bdist_wheel --release --universal"
+        create_wheel_cmd = "python setup.py bdist_wheel --release"
```

Comment:
Does this change affect the binaries in any way?

Reply:
Yes and that's my intent

`--universal` flag is to make sure that binaries work for both python 2 and 3. The minimum version of python we support is 3.8 so might as well just do this as well